.travis.yml

```diff
@@ -12,6 +12,16 @@ matrix:
         - BUILD_APIDOC=true
         # Auth token to push API documentation
         - secure: "JKeXk8p65hodb12PVRST6A90swsNubc+46EbSJGSghldIxbFWLBAlwU+KLeOMO4V0veu6k4lnMa50V0UYFZmoUsS6W0aL5Ybo98SpzXHiNLOmOluoqJoF9TBsOTCCRFbWbccgJyVEtulgRcdml96naS51lq9Sw/VO/N3Z472304="
+    - os: linix
+      language: python
+      python: 2.7.14
+      install:
+        - pip install pytest
```

Comment:
I doubt any `pytest` version is fine. Please use an explicit version marker like `pip install pytest==2.7.3`.